horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/Util.java

```diff
@@ -431,6 +431,11 @@ public static Object findJsonPath(JsonNode input, String jsonPath) {
         try {
             JsonPath path = JsonPath.compile(jsonPath);
             Object obj = ctx.read(path);
+            if (obj instanceof ArrayNode) {
+                if (((ArrayNode) obj).size() == 1) {
+                    obj = ((ArrayNode) obj).get(0);
+                }
+            }
```

Comment:
This could be simplified with something like this:

```suggestion
            if (obj instanceof ArrayNode arr) {
                if (arr.size() == 1) {
                    obj = arr.get(0);
                }
            }
```